components/eamxx/src/diagnostics/field_at_height.cpp

```diff
@@ -62,8 +68,8 @@ set_grids (const std::shared_ptr<const GridsManager> grids_manager)
   add_field<Required>(m_field_name,gname);
 
   // We don't know yet which one we need
-  add_field<Required>("z_mid",gname);
-  add_field<Required>("z_int",gname);
+  add_field<Required>(m_surf_ref+"_mid",gname);
```

Comment:
I don't think we need to add both mid and int as deps, right?

Reply:
I think we do, since we don't what layout the source field has until we get to initialization.  Unless there has been an update in how we can access fields in the field manager since this diagnostic was written...

Reply:
Ah, yes. I didn't realize that the request for the input field was created just with field/grid name, without layout.